src/Controller/ExplorerController.php

```diff
@@ -65,7 +65,7 @@ public static function create(ContainerInterface $container) {
    *
    * @codeCoverageIgnore
    */
-  public function __construct(UrlGeneratorInterface $urlGenerator, Introspection $introspection, SchemaPluginManager $pluginManager) {
+  public final function __construct(UrlGeneratorInterface $urlGenerator, Introspection $introspection, SchemaPluginManager $pluginManager) {
```

Comment:
in drupal we don't have final constructors. What is the error message from phstan that you get for this?

Reply:
Unsafe usage of new static().                                                                                                                 
         💡 Consider making the class or the constructor final.

Reply:
ah yes, that is not a best practice in Druapl, sow e should not fix that.
Use this exception:
```
# new static() is a best practice in Drupal, so we cannot fix that.
- "#^Unsafe usage of new static\\(\\)\\.$#"
```